GenderPayGap.WebUI/Views/Register/ReviewAddress.cshtml

```diff
@@ -1,35 +1,59 @@
-    <ol>
-        <li style="display: block; padding-bottom: 0; padding-top: 0.75em;">
-            &lt;
-            @Html.ActionLink("Back to Pending Registrations", "PendingRegistrations", "Admin")
-        </li>
-    </ol>
-</div>
-@using (Html.BeginForm("ReviewRequest"))
-{
-    @Html.AntiForgeryToken()
+@{
+    ViewBag.Title = "Review address change - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
 
-    @Html.HiddenFor(model => model.SectorType)
-    @Html.HiddenFor(model => model.CompanyNumber)
-    @Html.HiddenFor(model => model.CharityNumber)
-    @Html.HiddenFor(model => model.MutualNumber)
-    @Html.HiddenFor(model => model.OtherName)
-    @Html.HiddenFor(model => model.OtherValue)
-    @Html.HiddenFor(model => model.ReviewCode)
-    @Html.HiddenFor(model => model.ManualAddress)
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("AdminHomePage", "AdminHomepage")
+            },
+            new CrumbViewModel
+            {
+                Text = "Pending Registrations",
+                Href = Url.Action("PendingRegistrations", "AdminPendingRegistrations")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.ContactFirstName + " " + Model.ContactLastName + " (" + Model.ContactJobTitle + ") - " + Model.OrganisationName
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
 
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            @await Html.CustomValidationSummaryAsync()
-            <h1 class="heading-large">
-                <span>Approve existing employer application - with change of address</span>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="POST" action="@Url.Action("ReviewRequest", "Register")">
+            @Html.AntiForgeryToken()
+            
+            @Html.HiddenFor(model => model.SectorType)
+            @Html.HiddenFor(model => model.CompanyNumber)
+            @Html.HiddenFor(model => model.CharityNumber)
+            @Html.HiddenFor(model => model.MutualNumber)
+            @Html.HiddenFor(model => model.OtherName)
+            @Html.HiddenFor(model => model.OtherValue)
+            @Html.HiddenFor(model => model.ReviewCode)
+            @Html.HiddenFor(model => model.ManualAddress)
+            
+            @(Html.GovUkErrorSummary())
+
+            <h1 class="govuk-heading-xl">
+                Approve existing employer application - with change of address
             </h1>
+            
             <table class="govuk-table">
                 <caption class="govuk-table__caption govuk-!-margin-bottom-2">Person registering to report and change address</caption>

```

Comment:
I really don't like this caption being the same font-size as the text in the `th`s - I feel it'd make more sense for it to be an h3 (rather than an h2, which would put it on the same level as "Similar organisations"). but I am not an accessibility expert and presumably there was a reason for it to be a `<caption>`?

Reply:
Yeah - the caption means that it's read out as a header to the table (and hence if a visually impaired user filters by headings they won't get this one since it only relates to the table content, not to a grouping of other data). This is the "accessible way" recommended by the Design System 😄